donate/payments/tasks.py

```diff
@@ -73,48 +73,39 @@ def mofo_donation_receipt_number_format(amount):
     "service": "payment_source",
 }
 
-# These are compared with the donating users locale, to tell the Acoustic API what language to send the email
```

Comment:
moved this array to languages.py, so we can use it in both this file and in base.py where we are setting defaults